src/js/view/controls/components/volumetooltip.js

```diff
@@ -25,10 +25,11 @@ export default class VolumeTooltip extends Tooltip {
             this.trigger('update', evt);
         }, this);
 
-        this.closeTooltipHandler = this.closeTooltip.bind(this);
-        this.openTooltipHandler = this.openTooltip.bind(this);
-        this.container.addEventListener('blur', this.closeTooltipHandler);
-        this.container.addEventListener('focus', this.openTooltipHandler);
+
+        this.uiOver = new UI(this.container)
+            .on('click', function() {}, this)
```

Comment:
why are we adding the click listener ?

Reply:
I believe it is because we don't want the tooltip to toggle state when clicked on. `this.uiOver` is on the `.jw-overlay`, but it's a child of the volume tooltip that `this.ui` is created on so adding an empty action when `this.uiOver` is clicked resets it because otherwise, the `this.ui` click handler would toggle the tooltip's open state.

Reply:
Add a comment to explain why the listener is added. The focus helper logic is in ui.js `interactStartHandler`.